README.md

```diff
@@ -176,902 +252,202 @@ Sample bands configuration file:
-                                [--spread-feed SPREAD_FEED]
-                                [--spread-feed-expiry SPREAD_FEED_EXPIRY]
-                                --eth-reserve ETH_RESERVE
-                                [--min-eth-balance MIN_ETH_BALANCE]
-                                --min-eth-deposit MIN_ETH_DEPOSIT
-                                --min-sai-deposit MIN_SAI_DEPOSIT
-                                [--gas-price GAS_PRICE] [--smart-gas-price]
-                                [--debug]
-
-optional arguments:
-  -h, --help            show this help message and exit
-  --rpc-host RPC_HOST   JSON-RPC host (default: `localhost')
-  --rpc-port RPC_PORT   JSON-RPC port (default: `8545')
-  --rpc-timeout RPC_TIMEOUT
-                        JSON-RPC timeout (in seconds, default: 10)
-  --eth-from ETH_FROM   Ethereum account from which to send transactions
-  --tub-address TUB_ADDRESS
-                        Ethereum address of the Tub contract
-  --idex-address IDEX_ADDRESS
-                        Ethereum address of the IDEX contract
-  --idex-api-server IDEX_API_SERVER
-                        Address of the IDEX API server (default:
-                        'https://api.idex.market')
-  --idex-timeout IDEX_TIMEOUT
-                        Timeout for accessing the IDEX API (in seconds,
-                        default: 9.5)
-  --config CONFIG       Bands configuration file
-  --price-feed PRICE_FEED
-                        Source of price feed
-  --price-feed-expiry PRICE_FEED_EXPIRY
-                        Maximum age of the price feed (in seconds, default:
-                        120)
-  --spread-feed SPREAD_FEED
-                        Source of spread feed
-  --spread-feed-expiry SPREAD_FEED_EXPIRY
-                        Maximum age of the spread feed (in seconds, default:
-                        3600)
-  --eth-reserve ETH_RESERVE
-                        Amount of ETH which will never be deposited so the
-                        keeper can cover gas
-  --min-eth-balance MIN_ETH_BALANCE
-                        Minimum ETH balance below which keeper will cease
-                        operation
-  --min-eth-deposit MIN_ETH_DEPOSIT
-                        Minimum amount of ETH that can be deposited in one
-                        transaction
-  --min-sai-deposit MIN_SAI_DEPOSIT
-                        Minimum amount of SAI that can be deposited in one
-                        transaction
-  --gas-price GAS_PRICE
-                        Gas price (in Wei)
-  --smart-gas-price     Use smart gas pricing strategy, based on the
-                        ethgasstation.info feed
-  --debug               Enable debug output
-```
-
-### Known limitations
+#!/bin/bash
+    
+    bin/oasis-market-maker-keeper \
+        --rpc-host 127.0.0.1 \
+        --rpc-port <port number> \
+        --rpc-timeout 10 \
+        --eth-from [address of your generated Ethereum account] \
+    		--eth-key ${ACCOUNT_KEY} \ 
```

Comment:
So I think it's a good idea for you to have the actual example string in here:
`-- eth-key "key_file=./keystore.json,pass_file=keystore.pass"`
@grandizzy can you verify this is correct ^^ ?

Reply:
It should look like this:
`--eth-key "key_file=./keystore.json,pass_file=keystore.pass"`

Reply:
Verified by @EdNoepel. Resolving conversation.